### DISCUSSION TITLE
sidebarOpen with icons only

### DISCUSSION THREAD

**Discussion by martimpassos** [ProjectMirador/mirador#3581] [Looking for examples]
To facilitate discovery of metadata, annotations and rights by users, but avoid taking up too much screen area, it would be helpful to default to the collapsed sidebar like here:

![image](https://user-images.githubusercontent.com/49573683/192552710-b18a40d1-8804-48e4-83b8-4d526cfcd0e7.png)

Maybe modify `sidebarOpen` to `sidebarBehavior` with `closed`, `expanded` and `collapsed` as arguments? Or rather a new boolean config `contentPanelOpen` to detach it from the former?

**Comment by marlo-longley** [comment]
@martimpassos hello! Unfortunately it seems like the Github discussions area is not very well tended. I didn't even know it existed til recently. I converted your idea to a ticket for better tracking: https://github.com/ProjectMirador/mirador/issues/4515 - thanks!
